sdk/openai/openai/src/models/models.ts

```diff
@@ -1728,6 +1728,18 @@ export interface EmbeddingsUsage {
   totalTokens: number;
 }
 
+/** The OpenAI error object. */
+export interface OpenAIErrorModel {
+  /** A human-readable representation of the error. */
+  message?: string;
+  /** The parameters of the error. */
+  param?: string;
+  /** Type of the error. */
+  type?: string;
+  /** The error code. */
+  code?: string;
+}
```

Comment:
I am seeing nulls returned by the service, should the type be modeled as follows instead?
```suggestion
export interface OpenAIErrorModel {
  /** A human-readable representation of the error. */
  message?: string;
  /** The parameters of the error. */
  param: string | null;
  /** Type of the error. */
  type: string | null;
  /** The error code. */
  code: string | null;
}
```

Or should we still keep things optional too?

Reply:
Yes, that makes more sense. I'll update the type to `null` and follow response returned by the service